### ISSUE TITLE
simpleCNN

### ISSUE THREAD

**Issue by rr-lau** [ShowLo/SimpleNet#1]
import torch
from torch import nn
from torch.utils.data import DataLoader
from torchvision import datasets
from torchvision import transforms

"""
	1. Data Loader
"""
# Choose a dataset -- CIFAR10 for example
dataset = datasets.CIFAR10(root='data', train=True, download=True)

# Set how the input images will be transformed
dataset.transform = transforms.Compose([
    transforms.ToTensor(),
    transforms.Normalize(mean=[0.491, 0.482, 0.447],
                         std=[0.247, 0.244, 0.262])
])

# Create a data loader
train_loader = DataLoader(dataset, batch_size=256, shuffle=True, num_workers=0)

# Show the shape of a batch.

print("The shape of one batch is {}".format((next(iter(train_loader)))[0].size()))

""" 	2. Model. 

The requirements are:
    Define the first convolutional layer with channel size = 5, kernel size = 3 and stride = 2, padding = 1.
    Define the second convolutional layer with channel size = 8, kernel size = 3 and stride = 1, padding = 1.
    Use max pooling layer with stride = 2 between the two convolution layers.
    Define the FC layer(s) and finally return a tensor with shape torch.Size([256, 10]). (Use torch.view to reshape tensors. You can try any number of FC layers).
    Use ReLU activation between any two layers.
"""

class SimpleCNN(nn.Module):
    def __init__(self):
        super(SimpleCNN, self).__init__()  # Call parent class's constructor
        
        #############################################################################
        # TODO: Define the model structure                                          #
        #############################################################################
        
        

        
        
        #############################################################################
        #                          END OF YOUR CODE                                 #
        #############################################################################
        
        def forward(self, x):
        #############################################################################
        # TODO: Define forward function                                             #
        #############################################################################
            
            
            

            
        #############################################################################
        #                          END OF YOUR CODE                                 #
        #############################################################################
        return y
    

model = SimpleCNN()

"""
	3. Define Loss and Optimizer
"""

#############################################################################
# TODO: Define the criterion to be Cross Entropy Loss.                      #
#       Define the optimizer to be SGD with momentum factor 0.9             #
#       and weight_decay 5e-4.                                              #
#############################################################################

# criterion = 

# optimizer =

#############################################################################
#                          END OF YOUR CODE                                 #
#############################################################################

"""
	4. Start training
"""

def train(epoch):
    model.train()  # Set the model to be in training mode
    for batch_index, (inputs, targets) in enumerate(train_loader):
        # Forward
        outputs = model(inputs)
        loss = criterion(outputs, targets)
        if batch_index % 10 == 0:
            print('epoch {}  batch {}/{}  loss {:.3f}'.format(
                epoch, batch_index, len(train_loader), loss.item()))

        # Backward
        optimizer.zero_grad()  # Set parameter gradients to zero
        loss.backward()        # Compute (or accumulate, actually) parameter gradients
        optimizer.step()       # Update the parameters

for epoch in range(1, 6):
    train(epoch)
